docs/classes/Automattic-Domain-Services-Command-Dns-Get.md

```diff
@@ -29,12 +29,6 @@ if ( $response->is_success() ) {
 ### Methods
 
 * public [__construct()](#method___construct)
-* public [get_client_txn_id()](#method_get_client_txn_id)
-* public [get_domain()](#method_get_domain)
-* public [get_name()](#method_get_name)
-* public [jsonSerialize()](#method_jsonSerialize)
-* public [set_client_txn_id()](#method_set_client_txn_id)
-* public [to_array()](#method_to_array)
```

Comment:
We discussed this topic a little [here](p1673881090256129/1673880508.835859-slack-C0BNMNMNG).

I think the general idea is that we don't want them to rely in whatever format we decide for the command serialization.
And also, since they will have the data before instantiating the command, they can still log it - with the original data - in the format they like.